Week2/MohammadDarwesh/mohammaddarwesh.js

```diff
@@ -0,0 +1,180 @@
+// for the first task i will do it with many ways just for practising.
+console.log ("Halo, dunia! ,Ciao, mondo! ,Hola, mundo!");
+
+let indonesian = "Halo, dunia!";
```

Comment:
I really liked how you tried to use different ways to print Hello world here. A point to note, however, is that you could have used `const` keyword instead of `let` while creating these variables. This is because you aren't re-assigning `indonesian`, `italian`, `spanish`, `helloWorld` etc variables anywhere.

If you remember from our class, always start using `const` for variables and switch to `let` if your program requires re-assigning a variable. 🙂

Reply:
yes it is my mistake .....i get it wrong in the class but now it is clear ....thank you.